Application/EdFi.Security.DataAccess/Models/ResourceClaimActionAuthorization.cs

```diff
@@ -3,21 +3,22 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
+using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
 namespace EdFi.Security.DataAccess.Models
 {
-    public class ResourceClaimAuthorizationMetadata
+    public class ResourceClaimActionAuthorization
     {
         [Key]
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
-        public int ResourceClaimAuthorizationStrategyId { get; set; }
+        public int ResourceClaimActionAuthorizationId { get; set; }
 
         [Required]
         public Action Action { get; set; }
 
-        public AuthorizationStrategy AuthorizationStrategy { get; set; }
+        public IReadOnlyList<ResourceClaimActionAuthorizationStrategies> ResourceClaimActionAuthorizationStrategies { get; set; }
```

Comment:
I'm wondering about the use of `IReadOnlyList<T>` here. All the other lists in the entity classes using `List<T>`. Maybe for our purposes (which is a read-only use case I believe) this is fine, but it's certainly inconsistent with the others and may lead to questions like this from developers in the future when there's no good reason.

I'd suggest changing this to `List<T>` to match the other properties already defined.

Reply:
fixed